scripts/run_short_benches_for_runtime.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+
+# Runs all benchmarks for all pallets, for a given runtime, provided by $1
+# Should be run on a reference machine to gain accurate benchmarks
+# current reference machine: https://github.com/paritytech/substrate/pull/5848
+
+runtime="$1"
+
+echo "[+] Running all benchmarks for $runtime"
+
+./artifacts/polkadot benchmark \
+    --chain "${runtime}-dev" \
+    --list |\
+  tail -n+2 |\
+  cut -d',' -f1 |\
+  uniq | \
+  grep -v frame_system > "${runtime}_pallets"
```

Comment:
```suggestion
  uniq > "${runtime}_pallets"
```
to be in sync with https://github.com/paritytech/polkadot/commit/5d9dc9bd1b95e9c64cfc547f45f0b30e7cc03097